### ISSUE TITLE
Example template repo

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/streamlit-template#1]
Heya @Matthew-Jennings,

Here is an example template repo. The app itself is contained all within the following file:

https://github.com/pymedphys/streamlit-template/blob/master/app.py

Everything else to support the testing infrastructure.

Cheers,
Simon

**Comment by Matthew-Jennings** [comment]
Wow, very cool!

**Comment by SimonBiggs** [comment]
Yeah :). I could imagine building just a few of these standalone apps and having them as templates for people to start their projects with.
